### PR TITLE
Customizing the notifier host

### PR DESCRIPTION
**Pull request by aminin**

### CODE REVIEW

File: src/Notifier.php [hunk 1]
```diff
@@ -190,4 +198,21 @@ public function notify($exc)
...
+     * Builds notices URL
+     *
+     * @param array $opt
+     * @return string
+     */
+    protected function buildNoticesURL(array $opt)
+    {
+        $schemeAndHost = $opt['host'];
+
+        if (!preg_match('~^https?://~i', $schemeAndHost)) {
```
Comment (vmihailenco) [inline src/Notifier.php:218]: Can you please avoid using RegExp here? It is called every time notice is sent. For example, you can use

```
function substr_startswith($haystack, $needle) {
    return substr($haystack, 0, strlen($needle)) === $needle;
}
```

or build URL once and cache it in the notifier.

**Comment by vmihailenco** [comment]
Left one note, but otherwise looks good!

**Comment by JuliaYatmanova** [comment]
I am an erbit user too. When do you merge this pull request?

**Comment by vmihailenco** [comment]
Right now, I did not know that @aminin addressed my comment.

@aminin thanks for adding this.

**Pull request merged**